core/domain/collection_services.py

```diff
@@ -735,6 +743,22 @@ def publish_collection_and_update_user_profiles(committer_id, col_id):
             contributor, contribution_time_msec)
 
 
+def unpublish_collection_update_user_profiles(committer_id, col_id):
```

Comment:
Nit: this should be ``unpublish_collection_and_update_user_profiles`` and probably go underneath ``publish_collection_and_update_user_profiles`` for consistency.

Also nit: s/``col_id``/``collection_id``